cmd/dlv/dlv_test.go

```diff
@@ -30,8 +30,10 @@ import (
 	"golang.org/x/tools/go/packages"
 )
 
-var testBackend string
-var ldFlags string
+var (
```

Comment:
Please undo the formatting noise (this, the declaration of dlvArgsLen, the change to unrelated comments, etc).

Reply:
@aarzilli formatter has been forcing it, will undo.